src/liboslexec/CMakeLists.txt

```diff
@@ -62,13 +62,13 @@ set ( liboslexec_target_srcs
    ../liboslnoise/wide/wide_gabor_hybrid_disabled
    ../liboslnoise/wide/wide_gabor_hybrid_enabled
    ../liboslnoise/wide/wide_gabor_isotropic_disabled
-   ../liboslnoise/wide/wide_gabor_isotropic_enabled   
```

Comment:
Can you revert all the whitespace changes here? I don't mind fixing that eventually, but I'd rather do it as a separate PR and do the git blame trick since nothing really changed.

Reply:
Fixed.